provision.sh

```diff
@@ -10,6 +16,19 @@ package() {
     sudo apt-get -y install "$@"
 }
 
+git_clone() {
+    # Clone git repo into directory, clearing directory if it exists
+    # Usage: git_clone URL DIR [GIT_CLONE OPTIONS...]
+
+    local url="$1"
+    shift
+    local dir="$1"
+    shift
+
+    sudo rm -rf "$dir"
```

Comment:
Yes. I introduced `sudo` here to solve a problem when trying to delete one of the build folders (inside capstone?). Some of the build files were owned by root.